src/ng-annotate-main.js

```diff
@@ -456,12 +455,6 @@ function stringify(ctx, arr, quot) {
     }).join(", ") + "]";
 }
 
-function parseExpressionOfType(str, type) {
```

Comment:
Was not being used by anything, and keeping it would require a different handling of parser